Description: Fix build failure: 'SYS_gettid' undeclared
 The SYS_gettid syscall is Linux specific. So only use it when it is defined.
Author: Michael Biebl <biebl@debian.org>
Bug: https://bugzilla.gnome.org/show_bug.cgi?id=664545
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=646886
diff --git a/src/goabackend/goalogging.c b/src/goabackend/goalogging.c
index 89657bf..99bc008 100644
--- a/src/goabackend/goalogging.c
+++ b/src/goabackend/goalogging.c
@@ -207,7 +207,11 @@ goa_log (GoaLogLevel     level,
       break;
     }
 
+#if defined(SYS_gettid)
   thread_str = g_strdup_printf ("%d", (gint) syscall (SYS_gettid));
+#else
+  thread_str = "unknown";
+#endif
   g_print ("%s%s%s.%03d:%s%s%s[%s]%s:%s%s%s:%s %s %s[%s, %s()]%s\n",
            _color_get (_COLOR_BOLD_ON), _color_get (_COLOR_FG_YELLOW), time_buf, (gint) now.tv_usec / 1000, _color_get (_COLOR_RESET),
            _color_get (_COLOR_FG_MAGENTA), _color_get (_COLOR_BOLD_ON), thread_str, _color_get (_COLOR_RESET),
